diff --git a/node_modules/stellar-sdk/types/dom-monkeypatch.d.ts b/node_modules/stellar-sdk/types/dom-monkeypatch.d.ts
index 1a2287e..d7c2f9f 100644
--- a/node_modules/stellar-sdk/types/dom-monkeypatch.d.ts
+++ b/node_modules/stellar-sdk/types/dom-monkeypatch.d.ts
@@ -31,9 +31,10 @@ interface EventSource extends EventTarget {
    * event stream is set to "include", and false otherwise.
    */
   readonly withCredentials: boolean;
-  readonly CLOSED: number;
-  readonly CONNECTING: number;
-  readonly OPEN: number;
+  readonly CLOSED: 2;
+  readonly CONNECTING: 0;
+  readonly OPEN: 1;
+
   close(): void;
   addEventListener<K extends keyof EventSourceEventMap>(
     type: K,
